opencodelists/tests/fixtures.py

```diff
@@ -637,6 +638,27 @@ def new_style_version(universe, request):
     return type(version).objects.get(pk=version.pk)
 
 
+@pytest.fixture
+def create_codelists(organisation):
```

Comment:
This is a just a drive-by comment so feel free to ignore, but I doesn't look like we're actually using the fixture properties here at all (i.e. the closure over `organisation`) because we always supply the `owner` argument. So I think the whole thing could just be a helper function. But maybe you've got plans to use it in other contexts later where the default organisation will be useful?

Reply:
Yes, I mostly added it to the fixtures like this to be more generally useful later, intending to use the default org in the tests, but then I passed the owner arg explicitly just to make it clearer.